tools/README.md

```diff
@@ -75,7 +75,147 @@ dtrace: system integrity protection is on, some features will not be available
  flush_start:1000   flush_end:1000
 ```
 
-## tracegw.d
+## perfdw.d
+This is a simple dtrace script that measures latency times for when a r/w/f
+job is sent over the network to each downstairs to when the ACK for that job
+is returned to the upstairs. Jobs are sorted by type (r/w/f) and by each
+downstairs client ID.
+```
+sudo dtrace -s tools/perfdw.d
+```
+
+Here is an example of how it might look:
+```
+final:crucible alan$ sudo dtrace -Z -s tools/perfdw.d
+dtrace: system integrity protection is on, some features will not be available
+
+dtrace: script 'tools/perfdw.d' matched 0 probes
+^C
+
+  flush downstairs client 1
```

Comment:
I think the output here may have changd?

Reply:
Updated!